src/App.js

```diff
@@ -1,26 +1,33 @@
 import React, { Component } from 'react';
 import './App.css';
-import movieData from './movie-data';
+import { getMovies, getMovie } from './apiCalls.js'
 import MovieContainer from './Movies/Movies';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      currentMovie: {active: false, id: null}
+      movies: [],
+      currentMovie: {active: false, id: null, film: {}}
     }
   }
 
+  componentDidMount() {
+    getMovies()
+    .then(movies => this.setState({ movies: movies.movies }))
+    .catch(error => alert(error.message))
+  }
+
   goBack = () => {
     this.setState({
       currentMovie: { active: false, id: null } })
   }
 
   enlargeCard = (id) => {
     const foundMovie = this.state.movies.find(movie => movie.id === id)
-    this.setState({ currentMovie: {active: true, id: id}});
-    console.log(foundMovie);
+    getMovie(foundMovie.id)
+    .then(movie => this.setState({ currentMovie : {active: true, id: (movie.movie.id), film: movie.movie}}))
```

Comment:
Also, since "active" is only ever true when the film key has an object with the movie data present, is there a way to get rid of this boolean and just check the film data's presence instead?